test/gtest/uct/ib/test_ib.cc

```diff
@@ -269,6 +269,32 @@ void test_uct_ib_with_specific_port::cleanup() {
     }
 }
 
+
+class test_uct_ib_roce: public test_uct_ib_with_specific_port {
+public:
+    void init() {
+        test_uct_ib_with_specific_port::init();
+        test_uct_ib::init();
+    }
+
+    void cleanup() {
+        test_uct_ib::cleanup();
+        test_uct_ib_with_specific_port::cleanup();
+    }
```

Comment:
maybe just inherit from test_ib?
i think ROCE_LOCAL_SUBNET would just be ignored on IB link layer